src/main/js/store/actions/distributionConfigs.js

```diff
@@ -206,11 +238,19 @@ export function testDistributionJob(config) {
         request.then((response) => {
             response.json()
                 .then((responseData) => {
-                    if (response.ok) {
+                    const defaultHandler = () => jobError({
+                        type: DISTRIBUTION_JOB_TEST_FAILURE,
+                        ...responseData
+                    });
+                    const handler = createErrorHandler(DISTRIBUTION_JOB_TEST_FAILURE, defaultHandler);
+                    if (responseData.errors && !Object.keys(responseData.errors).length) {
                         dispatch(testJobSuccess(responseData.message));
```

Comment:
It seems odd to me that we are checking for values in the "errors" field and then passing along the "message" field.  Is this check  looking for errors? or is it looking for the absence of errors?

Reply:
looking for the absence of errors.